review

```diff
@@ -27,47 +27,11 @@ if [ "x$TOUK_BASH_HOME" = "x" ]; then
 fi
 
 # Includes
-. "$TOUK_BASH_HOME/touk-bash-git.sh"
-. "$TOUK_BASH_HOME/touk-bash-macos.sh"
+. "$TOUK_BASH_HOME/touk-bash-gerrit.sh"
```

Comment:
I prefere to include gerrit, git and macos alltogether. They should be more or less independent.

Reply:
My thought process was as follows: you can't use touk-bash-gerrit without touk-bash-git (and you do actually need the include in order for it to work), so as a consumer of touk-bash-gerrit you might just as well expect it to deliver all the functionality of touk-bash-git. That's the logic I was following.
Perhaps that makes things a bit more tangled up and non-obvious – since you're including just one script and other are included as well – but it seems cleaner to me to have _one source of truth_ instead of multiple (that's how I see it when the same script is included over and over).

Let me know whatcha think. 😎